server/src/main/java/org/opensearch/cluster/decommission/DecommissionService.java

```diff
@@ -117,7 +118,8 @@ private void setForcedAwarenessAttributes(Settings forceSettings) {
      */
     public void startDecommissionAction(
         final DecommissionAttribute decommissionAttribute,
-        final ActionListener<ClusterStateUpdateResponse> listener
+        final ActionListener<ClusterStateUpdateResponse> listener,
+        final TimeValue timeOutForNodeDraining
```

Comment:
Accept the entire request object here so that we don't always need to manually update the method here and just modifying the object will serve the purpose